client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementPublicationAndVoting.vue

```diff
@@ -457,19 +467,19 @@ export default {
   mounted() {
     this.$on('showUpdateForm', (index) => {
       for (const key in this.formFields) {
-        this.formFields[key] = this.votes[index]['attributes'][key]
+        this.formFields[key] = Object.values(this.votes)[index]['attributes'][key]
```

Comment:
This throws an error on editing

Reply:
do we receive the id then and not the index? Could it be, that its sometimes the Id and sometimes the index?

Reply:
The index is the id in the votes object. Like this: `{ voteId1: { name: 'Test', postcode: '12345', ..}, voteId2: { name: 'test2',..}}`

Reply:
The editable list uses the id as index and gives that back

Reply:
Done here [e616b66](https://github.com/demos-europe/demosplan-core/pull/3996/commits/e616b66bd1a1eb720cdaa48a6caf48e54817487c)